src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/StegService.kt

```diff
@@ -15,7 +16,7 @@ class StegService(
 ) {
 
     @Transactional
-    fun utførSteg(behandlingId: Long, behandledeSteg: BehandlingSteg) {
+    fun utførSteg(behandlingId: Long, behandledeSteg: BehandlingSteg, behandlingStegDto: BehandlingStegDto? = null) {
```

Comment:
Vil man noen gang kalle utførSteg fra en ikke-controller? Altså tasks eller noe sånt?
Da blir det kanskje litt rart at man krever en Dto for å kjøre utfør steg?

Reply:
behandlingStegDto kommer kun når frontend sender noen data som lagres i backend. Derfor er det et optional parameter